canonical-web-and-design/canonical.com
# Add secondary nav for /partners

## Done

- Add secondary nav for /partners pages
- Remove redirects to partners.ubuntu.com
- Replace some partners.ubuntu.com links to local partners links

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8002/partners
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See the secondary nav is visible and works fine


## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/2590
